code/game/data_huds.dm

```diff
@@ -415,6 +415,56 @@
 		else
 			holder.icon_state = ""
 
+/*~~~~~~~~~~~~
+	Circutry!
+~~~~~~~~~~~~~*/
+/obj/item/device/electronic_assembly/proc/diag_hud_set_circuithealth(hide = FALSE)
+	var/image/holder = hud_list[DIAG_CIRCUIT_HUD]
+	var/icon/I = icon(icon, icon_state, dir)
+	holder.pixel_y = I.Height() - world.icon_size
+	if((!isturf(loc))|hide) //if not on the ground dont show overlay
```

Comment:
```||```

Reply:
for my own future reference why || over |?

Reply:
```||``` is logical or, working worth truthy/falsy values.
```|``` is binary or, for working with bitfields and bitflags.

Reply:
thanks!